<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter</class>
 <widget class="QWidget" name="filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="filter_val"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="filter_btn">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select Criteria</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="criteria">
     <item>
      <widget class="QRadioButton" name="rdo_code">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Code</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_date">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_item">
       <property name="text">
        <string>Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_type">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="date">
     <item>
      <widget class="QLabel" name="from_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="start_date">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="to_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="end_date">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="type1">
     <item>
      <widget class="QRadioButton" name="rdo_export">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_import">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_ie">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="type2">
     <item>
      <widget class="QRadioButton" name="rdo_domestic">
       <property name="text">
        <string>Domestic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_Foreign">
       <property name="text">
        <string>Foreign</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdo_df">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="close_btn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTableWidget" name="data_table"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
